src/sub_component.py

```diff
@@ -0,0 +1,41 @@
+from talon import Context, Module
```

Comment:
rename to `sub_token_type` or `sub_token_range` or `sub_token_scope`.  And then let's use that for the capture / naming in file as well

Reply:
I guess `sub_component` is ok but I think the other options are clearer.  Or possibly `token_subcomponent`

Reply:
I think we can take this at the same time we actually update subtoken to word. Then we can actually update the five name, the capture and the list.